version/1/4/hi/code_of_conduct.md

```diff
@@ -0,0 +1,74 @@
+आचार संहिता # योगदानकर्ता वाचा
+
+## हमारी प्रतिज्ञा
+
+एक खुला और स्वागत करते वातावरण को बढ़ावा देने के हित में, हम के रूप में
+योगदानकर्ताओं और देखरेख हमारी परियोजना में भागीदारी बनाने के लिए प्रतिज्ञा और
```

Comment:
Maintainer => अनुरक्षक
So this becomes हम अर्थात  योगदानकर्ताओं और अनुरक्षक प्रतिज्ञा करते हैं की इस परियोजना एवं समुदाय में भाग लेना, एक उत्पीड़न मुक्त अनुभव रहेगा, बिना उम्र, शारीरिक आकार, विकलांगता, जातीयता, लिंग पहचान एवं अभिव्यक्ति, अनुभव का स्तर, राष्ट्रीयता, व्यक्तिगत उपस्थिति, जाति, धर्म, या यौन पहचान और उन्मुखीकरण के परवाह किये I